packages/drivers/http/README.md

```diff
@@ -14,13 +14,15 @@ export const login = (store, action, drivers) => {
 
   // TODOS is the context for this request
   // You can then catch events like `/@@http\/TODOS>.*/
-  http('TODOS')
+  // You have the possibility to add more context with an object on second parameters
```

Comment:
I prefer that you add a new paragraph and explain WHY we should (or not) context object

Reply:
And how